sdk/nodejs/policy/protoutil.ts

```diff
@@ -174,33 +175,33 @@ export function makeAnalyzeResponse(ds: Diagnostic[]) {
 /** @internal */
 export function mapEnforcementLevel(el: EnforcementLevel) {
     switch (el) {
-        case "advisory":
-            return analyzerproto.EnforcementLevel.ADVISORY;
-        case "mandatory":
-            return analyzerproto.EnforcementLevel.MANDATORY;
-        case "disabled":
-            return analyzerproto.EnforcementLevel.DISABLED;
-        default:
-            throw new UnknownEnforcementLevelError(el);
+    case "advisory":
```

Comment:
Can we configure `eslint` to prefer the extra indentation?

Most (all?) of our existing TypeScript uses the following indentation:

```typescript
switch (i) {
    case 1:
        ...
    case n:
        ...
}
```

Rather than:

```typescript
switch (i) {
case 1:
    ...
case n:
    ...
}
```